### PR TITLE
Soulrrrrr: Changed topic to contigious array

### PR DESCRIPTION
**Pull request by soulrrrrr**
After last time discussed with professor, I found out that the columnar array is nearly unable to work, so I decided to change the topic to contigious array.

---
A good PR should include the following parts:

- [ ] A clear title
- [ ] A meaningful message for PR, as well as its commits
- [ ] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [ ] excluding any irrelevant files, such as binaries, text files, or dot files
- [ ] passing tests/CI

### CODE REVIEW

**Comment by soulrrrrr** [comment]
Sure. Changed the file.

File: project/README.rst [hunk 1]
```diff
@@ -210,7 +210,7 @@ Time Table Session 3
   * - 27
     - 14:12 - 14:24
     - `soulrrrrr <https://github.com/soulrrrrr>`__
-    - `Columnar Array <soulrrrrr/README.rst>`__:
+    - `Contigious Array <soulrrrrr/README.rst>`__:
```
Comment (yungyuc) [inline project/README.rst:213]: It would be good to fix the typo.  Could you please also update the row in the timetable sessions 3?

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: project/README.rst (hunk 1 above)
Comment (soulrrrrr) [reply project/README.rst:213]: Typo fixed. Isn't this row the row in the timetable sessions 3? Is there any other place need to be modified?

Comment (yungyuc) [reply project/README.rst]: YOu are right.  I misread.

**Review by yungyuc — APPROVED** [review]

**Comment by yungyuc** [comment]
LGTM.  Thanks for the quick fix.

**Pull request merged**